### ISSUE TITLE
Click to copy url callout missing focus ring

### ISSUE THREAD

**Issue by weyert** [kentcdodds/remix-kentcdodds#126]
The `Click to Copy url`-button as part of an article item on the page doesn't have a focus ring like the other buttons throughout the page. This is how the button looks like when you tabbed to the button: 

![image](https://user-images.githubusercontent.com/7049/134374219-55f7e82c-c3f9-441c-8f06-b497d4e8d715.png)

Looks like the ring colour is white which gets lost in the white background  of the button. Once you know the ring colour is white you can see the small change in dimensions.
